_docs/example-collections.md

```diff
@@ -595,11 +595,11 @@ CODE_EXAMPLE_END
     }],
     "payin_methods": [{
         "type": "GHS::Mobile",
+        "ux_flow": "ussd_popup",
         "in_details": {
-            "phone_number": "+256301234567",
-            "send_instructions": true
+            "phone_number": "+2339999999", // In international format
+            "mobile_provider": "vodafone" // Mandatory. One of 'airtel', 'tigo', 'mtn', 'vodafone'
```

Comment:
Did we use the code example generator to generate these examples from the pseudocode above?

Reply:
Do note this has generated string literals instead of enum values, so they definitely look wrong. Please fix the pseudocode above, and run the code generator at https://github.com/transferzero/api-documentation/blob/master/generate_sdk_examples.rb